src/__tests__/http-test.js

```diff
@@ -75,11 +76,13 @@ function urlString(urlParams?: { [param: string]: string, ... }): string {
   return string;
 }
 
-[
+const implementations: Array<[(...args: Array<any>) => any, string]> = [
```

Comment:
This code was a constant source of frustration for me and now it even introduces bunch of any types.
So I just rewrote it in #631 
Can you please rebase?